### PR TITLE
Operating a node

### PR DESCRIPTION
**Pull request by MishaBoar**
This is a relatively complete first iteration of the guide for operating a node. Future versions can expand on some points, but this should be good to go for the time being.

### CODE REVIEW

**Comment by MishaBoar** [comment]
Probably the output of the "help" command can be omitted from the article.

File: content/dogepedia/how-tos/operating-a-node/index.md [hunk 1]
```diff
@@ -0,0 +1,495 @@
...
+In both these examples, only incoming connections are enabled.
+
+If you are using Dogecoin CLI, you can check the total number of connections by using the *getconnectiontotal* command:
+
+```console
+shibetoshi:~$ dogecoin-cli getconnectioncount
+12
+```
+
+In this case, since there are more than 8 active connection, we know that the node is already allowing incoming connections.
```
Comment (RichieJohn) [inline content/dogepedia/how-tos/operating-a-node/index.md:224]: change "connection" to "connections" (i.e., pluralise). :)

**Review by RichieJohn — COMMENTED** [review]
Much wow. Amazing, MishaBoar. Just a tiny suggestion from me (see line 224).

File: content/dogepedia/how-tos/operating-a-node/index.md (hunk 1 above)
Comment (MishaBoar) [reply content/dogepedia/how-tos/operating-a-node/index.md]: Done! Thanks @RichieJohn!

**Review by RichieJohn — APPROVED** [review]

**Pull request merged**